src/util/hass-util.html

```diff
@@ -385,6 +385,8 @@
     case 'safety':
     case 'smoke':
       return activated ? 'mdi:verified' : 'mdi:alert';
+    case 'plug':
+      return activated ? 'mdi:power-plug-off' : 'mdi:power-plug';
```

Comment:
This is reverse. When it is activated it should show plugged.

Reply:
This whole function has a reversed name:
`var activated = state.state && state.state === 'off';`

Reply:
that's weird